### PR TITLE
Remove pkg/scheduler/volumebinder.

### PR DESCRIPTION
**Pull request by TLIM023**
**What type of PR is this?**
/kind cleanup
/sig scheduling

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

None

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

**Comment by haosdent** [comment]
@TLIM023 seems cla haven't sign. Which email are you using to sign it?

**Comment by ahg-g** [comment]
/approve 

for the scheduler changes.

**Comment by haosdent** [comment]
@TLIM023 forgot to use correct email?

**Comment by haosdent** [comment]
Commit message also incorrect

```
resolve merge conflict
```

**Comment by haosdent** [comment]
```
ERROR(all) pkg/scheduler/scheduler_test.go:602:2: registry declared but not used
ERROR(all) pkg/scheduler/scheduler_test.go:313:5: unknown field client in struct literal
exit status 1
!!! Type Check has failed. This may cause cross platform build failures.
!!! Please see https://git.k8s.io/kubernetes/test/typecheck for more information.
```

@TLIM023

**Comment by haosdent** [comment]
@TLIM023 can squash to 1 commit?

**Comment by ahg-g** [comment]
/cc @msau42 

Needs Sig storage approval

**Comment by Huang-Wei** [comment]
/cc @janetkuo 
Could you review the controller-related changes?

File: pkg/controller/.import-restrictions [hunk 1]
```diff
@@ -270,7 +270,6 @@
         "k8s.io/kubernetes/pkg/security/apparmor",
         "k8s.io/kubernetes/pkg/util/parsers",
         "k8s.io/kubernetes/pkg/fieldpath",
-        "k8s.io/kubernetes/pkg/scheduler/volumebinder",
```
Comment (ahg-g) [inline pkg/controller/.import-restrictions]: please revert this line, https://github.com/kubernetes/kubernetes/pull/83526 is in flight and will do this cleanup.

Comment (haosdent) [reply pkg/controller/.import-restrictions]: he would update soon.

Comment (oomichi) [reply pkg/controller/.import-restrictions]: https://github.com/kubernetes/kubernetes/pull/83526 has been merged after the above review.
I guess this PR should be updated based on the latest one, right?

**Comment by angao** [comment]
I think this pr could be closed since [refactor volume binder](https://github.com/kubernetes/kubernetes/pull/88401) has been merged

**Comment by alculquicondor** [comment]
Can we close this PR?

**Pull request closed without merging**